test/api/controllers/readController.test.js

```diff
@@ -155,9 +155,9 @@ describe('Test: read controller', function () {
           should(realtime).be.true();
           should(stored).be.true();
           should(result.data.body.type).be.exactly('all');
-          should(result.data.body.collections).not.be.undefined().and.be.an.Object();
-          should(result.data.body.collections.stored).not.be.undefined().and.be.an.Array();
-          should(result.data.body.collections.realtime).not.be.undefined().and.be.an.Array();
+          should(result.data.body.collections).not.be.eql(undefined).and.be.an.Object();
+          should(result.data.body.collections.stored).not.be.eql(undefined).and.be.an.Array();
+          should(result.data.body.collections.realtime).not.be.eql(undefined).and.be.an.Array();
           should(result.data.body.collections.stored.sort()).match(['foo']);
```

Comment:
Why? [Undefined is a valid should assertion](https://shouldjs.github.io/#assertion-undefined)